From: Kevin Hilman <khilman@mvista.com>
Subject: [PATCH -rt] ARM: compile fix for event tracing

The cycles/usecs conversion macros should be dependent on
CONFIG_EVENT_TRACE instead of CONFIG_LATENCY_TIMING.

Signed-off-by: Kevin Hilman <khilman@mvista.com>

---
 include/asm-arm/timex.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.24-rt1/include/asm-arm/timex.h
===================================================================
--- linux-2.6.24-rt1.orig/include/asm-arm/timex.h	2008-01-25 15:07:14.000000000 -0500
+++ linux-2.6.24-rt1/include/asm-arm/timex.h	2008-01-25 15:08:31.000000000 -0500
@@ -18,7 +18,7 @@ typedef unsigned long cycles_t;
 
 #ifndef mach_read_cycles
  #define mach_read_cycles() (0)
-#ifdef CONFIG_LATENCY_TIMING
+#ifdef CONFIG_EVENT_TRACE
  #define mach_cycles_to_usecs(d) (d)
  #define mach_usecs_to_cycles(d) (d)
 #endif
